detection_rules/devtools.py

```diff
@@ -816,13 +830,19 @@ def raw_permalink(raw_link):
 @dev_group.command('trim-version-lock')
 @click.argument('stack_version')
 @click.option('--dry-run', is_flag=True, help='Print the changes rather than saving the file')
-def trim_version_lock(stack_version: str, dry_run: bool):
+@click.pass_context
```

Comment:
At some point we should consider whether we should move these commands or duplicate them into DAC-friendlier versions since devtools is intended for internal management. Its less sustainable to add exceptions like this everywhere